docs/modeling-your-data/running-data-models-via-snowplow-bdp/standard-models/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Standard models"
```

Comment:
Same as above — could you add a more descriptive title and move this to `sidebar_label`?

Reply:
Oh, and a 🆕 in the label